src/java-runtime/index.ts

```diff
@@ -9,6 +9,51 @@ import * as pathExists from "path-exists";
 import * as request from "request-promise-native";
 import findJavaHome = require("find-java-home");
 import architecture = require("arch");
+import { loadTextFromFile, getExtensionContext } from "../utils";
+
+let javaRuntimeView: vscode.WebviewPanel | undefined;
+
+export async function javaRuntimeCmdHandler(context: vscode.ExtensionContext, operationId: string) {
+  if (javaRuntimeView) {
+    javaRuntimeView.reveal();
+    return;
+  }
+
+  javaRuntimeView = vscode.window.createWebviewPanel("java.runtime", "Java Runtime", {
+    viewColumn: vscode.ViewColumn.One,
+  }, {
+    enableScripts: true,
+    enableCommandUris: true,
+    retainContextWhenHidden: true
+  });
+
+  await initializeJavaRuntimeView(context, javaRuntimeView, onDidDisposeWebviewPanel);
```

Comment:
`javaRuntimeView` is defined at line 14, then why we still need to pass it as a method arg?

Reply:
javaRuntimeView is defined outside but it does not stop the initializeJavaRuntimeView being implemented in a functional fashion. We try to only work with parameters and local variables.